packages/synthetics-sdk-broken-links/src/link_utils.ts

```diff
@@ -303,6 +335,37 @@ export function sanitizeObjectName(
     .replace(/\s+/g, '_'); // Replace one or more spaces with underscores
 }
 
+export function getStoragePathToExecution(
+  storageParams: StorageParameters,
+  options: BrokenLinksResultV1_BrokenLinkCheckerOptions
+) {
+  try {
+    const storageLocation = options.screenshot_options!.storage_location;
+    let writeDestination = '';
+
+    // extract folder name for a given storage location. If there is no '/'
+    // present then the storageLocation is just a folder
+    const firstSlashIndex = storageLocation.indexOf('/');
+    if (firstSlashIndex !== -1)
+      writeDestination = storageLocation.substring(firstSlashIndex + 1);
+
+    // Ensure writeDestination ends with a slash for proper path joining
+    if (writeDestination && !writeDestination.endsWith('/')) {
+      writeDestination += '/';
+    }
+
+    writeDestination = path.join(
```

Comment:
Do we want a slash between checkId and executionId?

Reply:
yup!

Reply:
I guess this question was getting at - if the path.join function adds a slash by are we doing it manually for write destination above?

Reply:
oh I see, yes I think that may be redundant